app/controllers/api/v1/tags_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::TagsController < Api::V1::BaseController
```

Comment:
You can ignore this one because the rest of the app uses it, but I bet there is a better way.